tests/e2e/deployment-test-cloud-function.sh

```diff
@@ -19,24 +19,23 @@ check_function_state()
 {
   FUNCTION=$1
   FUNCTION_DESCRIBE=$(gcloud functions describe "$FUNCTION"  --format="json")
-  FUNCTION_STATE=$(echo "$FUNCTION_DESCRIBE" | jq -r '.status')
+  FUNCTION_STATE=$(echo "$FUNCTION_DESCRIBE" | "$TEST_JQ" -r '.status')
   if [[ "${FUNCTION_STATE}" != *"ACTIVE"* ]]; then
     return 1
   fi
   return 0
 }
 
-install_yq
 gcloud config set project "${GCP_PROJECT_ID}"
 create_sample_app
 
 # Run cloud function deployment.
 rm -rf ./e2e_test
-mkdir -p ./e2e_test/gcp_iam_roles
-cp ./scripts/lib.sh ./e2e_test/lib.sh
-cp ./scripts/setup.sh ./e2e_test/setup.sh
-cp dynatrace-gcp-function.zip ./e2e_test/dynatrace-gcp-function.zip
-cp ./gcp_iam_roles/dynatrace-gcp-function-metrics-role.yaml ./e2e_test/gcp_iam_roles/
+mkdir ./e2e_test
+unzip -d ./e2e_test ./artefacts/function-deployment-package.zip
+cp ./artefacts/dynatrace-gcp-function.zip ./e2e_test
```

Comment:
why you extracting to e2e_test and then copy this zip to the same directory?

Reply:
function-deployment-package.zip doesn't contain dynatrace-gcp-function.zip. e2e tests assumes local zip file (git release is not available for download)